AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -630,7 +660,7 @@ - (NSUInteger)dataController:(ASDataController *)dataController rowsInSection:(N
   return [_asyncDataSource collectionView:self numberOfItemsInSection:section];
 }
 
-- (NSUInteger)dataControllerNumberOfSections:(ASDataController *)dataController {
+- (NSUInteger)numberOfSectionsInDataController:(ASDataController *)dataController {
```

Comment:
Nice, definitely like this.  The original API had a couple slightly off names from the original engineer :)